config/checkstyle-input-suppressions.xml

```diff
@@ -843,6 +843,8 @@
             files="checks[\\/]javadoc[\\/]javadocmissingwhitespaceafterasterisk[\\/]InputJavadocMissingWhitespaceAfterAsteriskValidWithTab.java"/>
   <suppress id="UnnecessaryOkComment"
             files="checks[\\/]javadoc[\\/]javadocmissingwhitespaceafterasterisk[\\/]InputJavadocMissingWhitespaceAfterAsteriskValidWithTab.java"/>
+  <suppress id="UnnecessaryOkComment"
+            files="checks[\\/]javadoc[\\/]javadocpackage[\\/]pkghtml[\\/]InputJavadocPackageHtmlIgnored2.java"/>
```

Comment:
@555vedant ,
The whole point of the issue is to pick a suppression and resolve it.